source/OptimaJet.DWKit.StarterApplication/Services/Contracts/IHangfireWrapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Hangfire;
+
+namespace OptimaJet.DWKit.StarterApplication.Services
+{
+    public interface IHangfireWrapper
+    {
+        IBackgroundJobClient BackgroundJobClient { get; }
```

Comment:
We will also need RecurringJob.

Reply:
Yes, this gives the framework for what I need for the first step.  You are right that something along that line will need to be added when we add the recurring job